task_22.py

```diff
@@ -0,0 +1,14 @@
+# 22. Write a Python program to check whether every element is a 3 or a 5 in a given array of integers.
```

Comment:
This script should return False in case array has elements different from 3 and 5. But not count elements. Please rewrite it.